docs/AboutUs.md

```diff
@@ -45,22 +45,14 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 * Role: Developer
 * Responsibilities: Data
 
-### Jean Doe
+### Loh Shi Jie, Elaine
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/elaineshijie.png" width="200px">
 
-[[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
```

Comment:
Is there supposed to be a link to your PPP?

Reply:
oh opps I added in in the next commit